templates/glossary.md

```diff
@@ -0,0 +1,331 @@
+# Lean Glossary
+
```

Comment:
Would it make sense to add a sentence or two about what this document is? Maybe note that these entries aren't intended to be definitions, but to clarify their typical usage in the Lean community/Zulip. I say this because a lot of the terms do have technical meanings that aren't precisely captured here.

Reply:
Yes definitely a good idea, done.